lib/proofer/vendor/mock.rb

```diff
@@ -46,6 +50,26 @@ def submit_phone(phone_number, session_id = nil)
         end
       end
 
+      # rubocop:disable MethodLength
+      def submit_state_id(state_id_data, session_id = nil)
+        if !SUPPORTED_STATES.include? state_id_data[:state_id_jurisdiction]
+          failed_confirmation(
+            MockResponse.new(session: session_id, reasons: ['invalid jurisdiction']),
+            state_id_jurisdiction: 'The jurisdiction could not be verified'
+          )
+        elsif state_id_data[:state_id_number] =~ /\A0*\z/
+          failed_confirmation(
+            MockResponse.new(session: session_id, reasons: ['invalid state id number']),
+            state_id_number: 'The state ID number could not be verified'
```

Comment:
Where are the state ID numbers?

Reply:
State ID numbers are obtained from the AAMVA API as well. I don't think there's state id mock data anywhere for the mock proofer but I might be wrong. We just went with the format being nine digits. Perhaps, it would help to add a comment here

Reply:
`NotImplemented` is raised if the `submit_state_id` is not implemented in the `Mock` class